src/main/java/feign/error/ErrorHandling.java

```diff
@@ -28,5 +30,18 @@
   Class<? extends Exception> defaultException() default NO_DEFAULT.class;
 
   final class NO_DEFAULT extends Exception {
+
+      /**
+       * For backward compatibility.
+       */
+      @Deprecated
```

Comment:
Why not straight out delete it? Just wondering if anyone outside of the annotation-error-decoder actually explicitly uses NO_DEFAULT... (which would be weird to me)

Reply:
No problem. I wasn't sure about the policy of introducing breaking changes. Removed this constructor entirely.